src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCopy.java

```diff
@@ -1768,8 +1766,6 @@ private void getDestinationMetadata() throws SQLServerException {
                 rs.close();
             if (null != stmt)
```

Comment:
why not try-with-resouce for this as well?

Reply:
This ResultSet 'rs' and Statement 'stmt' objects are created based on IF condition above, where, Statement might not even be created in 1 condition, hence the finally block is kept as it is.